atom/catom.pyi

```diff
@@ -68,11 +73,19 @@ class Member(Generic[T, S]):
     def __delete__(self, instance: Atom) -> None: ...
     def tag(self: M, **kwargs) -> M: ...
     def clone(self: M) -> M: ...
-    def add_static_observer(self, observer: str | Callable[..., None]) -> Any: ...
+    def add_static_observer(
+        self,
+        observer: str | Callable[..., None],
```

Comment:
Here we can do better following what I did for observe in atom.py. sorry I forgot to do the same update here.